### ISSUE TITLE
Make ordering of candidates more random

### ISSUE THREAD

**Issue by SK53** [russss/solartagger#1]
Using the OSM Id tends to mean that lots of adjacent panels appear in sequence. When such panels are predominantly of identical size there's an increased likelihood that "odd ones out" get tagged with an incorrect number of modules. Because users may get used to a pattern this may occur across users too. 

I'd suggest hashing the OSM Id in some way in the ORDER BY clause of the SELECT to reduce this likelihood.

**Comment by russss** [comment]
Yeah, this is a bit tricky because I decided to show newer OSM IDs first, as they tend to have better imagery alignment than older ones, so there's a better chance of success with newer IDs.

I also quite like being able to tag nearby nodes at the same time, but I get where you're coming from with the errors.
